gmsh_utils/gmsh_IO.py

```diff
@@ -158,12 +158,16 @@ def create_surface(self, line_ids: Union[List[int], npt.NDArray[np.int_]], name_
         curve_loop_id = gmsh.model.occ.addCurveLoop(line_ids)
         surface_id: int = gmsh.model.occ.addPlaneSurface([curve_loop_id])
         surface_ndim = 2
-        gmsh.model.setPhysicalName(surface_ndim, surface_id, name_label)
+
+        # only add physical group if name label is not empty
+        if name_label != "":
+            gmsh.model.setPhysicalName(surface_ndim, surface_id, name_label)
```

Comment:
can you check if "setPhysicalName" is required?, since you also add the name in "addPhysicalGroup"